snake.py

```diff
@@ -38,15 +44,13 @@ def drawGrid(w, rows, surface):
         x = x + sizeBtwn
         y = y + sizeBtwn
 
-        pygame.draw.line(surface, (255,255,255), (x,0),(x,w))
-        pygame.draw.line(surface, (255,255,255), (0,y),(w,y))
-
-
+        pygame.draw.line(surface, (255, 255, 255), (x, 0), (x, w))
+        pygame.draw.line(surface, (255, 255, 255), (0, y), (w, y))
 
 
 def redrawWindow(surface):
     global rows, width
-    win.fill((0,0,0))
+    surface.fill((0, 0, 0))

```

Comment:
here you are passing in your window object as "surface" and not as win. So you have to refer to that object by the parameter name you give it when you define the function.

Later when you call the function you pass in win for the surface parameter, but the function doesn't care about that, it cares about the parameter name

Reply:
Wow, thank you times infinity bro. So this is Object oriented programming? What does it mean to be Pythonic?